src/Infrastructure/MailTemplateFilenameTraversable.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MailTemplateFilenameTraversable implements \IteratorAggregate {
```

Comment:
I reckon we agreed on using `use` statements instead of inline absolute namespaces